build.gradle

```diff
@@ -17,6 +17,9 @@
  */
 
 plugins {
+    id 'com.github.spotbugs-base'
+    id 'com.github.johnrengelman.shadow'
+    id 'de.undercouch.download'
```

Comment:
I think we can skip the download plugin as well since we don't have native code, which requires the checkstyle plugin that uses the download plugin.